examples/script-processor.mjs

```diff
@@ -0,0 +1,27 @@
+import { AudioContext, GainNode, OscillatorNode, ScriptProcessorNode } from '../index.mjs';
+import { kNapiObj } from '../js/lib/symbols.js'
+
+const latencyHint = process.env.WEB_AUDIO_LATENCY === 'playback' ? 'playback' : 'interactive';
+const audioContext = new AudioContext({ latencyHint });
+
+const sine = new OscillatorNode(audioContext);
+sine.frequency.value = 200;
+
+const scriptProcessor = new ScriptProcessorNode(audioContext);
+const buffer = new Float32Array(scriptProcessor.bufferSize);
+scriptProcessor.addEventListener('audioprocess', e => {
```

Comment:
We should be able to modify the samples without an intermediate buffer no?
First just copy the input to output, then something like

    const channelData = audioBuffer.getChannelData(channel);

    for (let sample = 0; sample < channelData.length; sample += 1) {
        channelData[sample] +=  Math.random() * 2 - 1;
    }

Reply:
Yes indeed, this works as well. I just didn't want to pile up possible issues for this first test as our `getChannelData` is not the most reliable thing for now, cf. #80